antarest/study/web/study_data_blueprint.py

```diff
@@ -1020,6 +1021,51 @@ def remove_constraint_term(
             study, binding_constraint_id, term_id
         )
 
+    @bp.get(
+        path="/studies/{uuid}/areas/{area_id}/hydro/allocation",
+        tags=[APITag.study_data],
+        summary="Get allocation options for a given area",
+        response_model_exclude_none=True,
```

Comment:
You can document the Swagger API by adding:

```python
response_model=AllocationFormFields,
```

Why using `response_model_exclude_none=True`?